modules/fundamental/src/sbom/model/mod.rs

```diff
@@ -118,7 +118,13 @@ pub struct SbomPackage {
     /// CPEs identifying the package
     pub cpe: Vec<String>,
     /// License info
-    pub licenses: Option<String>,
+    pub licenses: Vec<LicenseBasicInfo>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, ToSchema, SimpleObject)]
+pub struct LicenseBasicInfo {
+    pub(crate) license_name: String,
```

Comment:
To my understanding this is being exported on the API. So why make this `pub(crate)`? I think it's ok to have this `pub`.